app/main/views/suppliers.py

```diff
@@ -343,6 +343,9 @@ def submit_company_summary():
 
 @main.route('/create-your-account', methods=['GET'])
 def create_your_account():
+    current_app.logger.info(
+        "suppliercreate: get create-your-account supplier_id:{}".format(
+            session['email_supplier_id']))
```

Comment:
Although, do we want it to error in that case and not log the get?  In which case we can leave it as is.  It will 500 though I think, which isn't nice.